File: CarRental-Source/CarRental/CarRental.Web/Models/AccountChangePasswordModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace CarRental.Web.Models
{
    public class AccountChangePasswordModel
    {
        public string LoginEmail { get; set; }
        public string OldPassword { get; set; }
        public string NewPassword { get; set; }
    }
}

File: CarRental-Source/CarRental/CarRental.Web/Bindings/Models/AccountChangePasswordModel.js
(function (cr) {
	let AccountChangePasswordModel = function () {
		let self = this;

		self.LoginEmail = ko.observable("");

		self.OldPassword = ko.observable("").extend({
			required: { message: 'Old password is required' },
			minLength: { message: 'Old password must be at least 6 characters', params: 6 }
		});

		self.NewPassword = ko.observable("").extend({
			required: { message: 'Old password is required' },
			minLength: { message: 'Old password must be at least 6 characters', params: 6 }
		});
	};
	cr.AccountChangePasswordModel = AccountChangePasswordModel;
}(window.CarRental))